### ISSUE TITLE
Regression: Popwin broken by "core: display warning buffers at the bottom of the screen"

### ISSUE THREAD

**Issue by aaronjensen** [syl20bnr/spacemacs#6220]
#### Description
popwin was broken by 1ebbe18b478945b0c1eacfd6a3a82b22c3fdbd87

I don't know how exactly, but before it, popwin worked, now it doesn't.

Of note is that `display-buffer-alist` is `nil` after the commit, but set to
`((popwin:display-buffer-condition popwin:display-buffer-action))` before the
commit. Likely the [`let`](https://github.com/syl20bnr/spacemacs/commit/1ebbe18b478945b0c1eacfd6a3a82b22c3fdbd87#diff-2d7025db7d7e8c7c4cc1dee8c1e90ba8R300) is screwing things up.

#### Reproduction guide
- Start Emacs
- Eval:

```elisp
(push '("*alchemist test report*"
        :position bottom :noselect t :dedicated t :stick t :height 25)
      popwin:special-display-config)
```

- Open an elixir test and `,tb`

*Observed behaviour:*
- Window opens using default window behavior, either on the right or bottom

*Expected behaviour:*
- Window pops at bottom as specified by `popwin:special-display-config`

#### System Info
- OS: darwin
- Emacs: 25.0.93.1
- Spacemacs: 0.105.20
- Spacemacs branch: nil (rev. 1ebbe18)
- Graphic display: t
- Distribution: spacemacs
- Editing style: hybrid
- Completion: helm
- Layers:
```elisp
(better-defaults spacemacs-layouts helm emacs-lisp markdown
                 (syntax-checking :variables syntax-checking-enable-tooltips nil)
                 (auto-completion :variables auto-completion-enable-sort-by-usage nil auto-completion-enable-snippets-in-popup nil)
                 erlang elixir git dash html clojure
                 (org :variables org-enable-github-support t)
                 colors osx github javascript react deft floobits ruby
                 (shell :variables shell-default-shell 'ansi-term shell-default-height 30 shell-default-position 'bottom)
                 spell-checking ranger version-control rcirc tmux evil-little-word auto-correct frame-geometry cleverparens-lispy)
```

**Comment by syl20bnr** [comment]
Oh yeah... I see, this is a bit tricky. It happens because popwin is initialized inside this let :-/
I'll see what I can do, thank you for the report.

**Comment by syl20bnr** [comment]
I moved the let at install time only, should fix the issue.

**Comment by aaronjensen** [comment]
Fixed, thanks!

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
